bin/dist

```diff
@@ -6,7 +6,7 @@ REV=$(git rev-parse --short HEAD)
 
 CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -ldflags "-X main.rev=$REV"
 
-OWNER=nytimes
+OWNER=${OWNER-nytimes}
```

Comment:
It seems like bash [also accepts this syntax](http://www.tldp.org/LDP/abs/html/parameter-substitution.html), however I will update it to be `OWNER=${OWNER:-nytimes}` to make it more explicit.